src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -37,6 +37,7 @@ public enum StandardActions implements Action {
     OPEN_URL(Localization.lang("Open URL or DOI"), IconTheme.JabRefIcons.WWW, KeyBinding.OPEN_URL_OR_DOI),
     SEARCH_SHORTSCIENCE(Localization.lang("Search ShortScience")),
     MERGE_WITH_FETCHED_ENTRY(Localization.lang("Get bibliographic data from %0", "DOI/ISBN/...")),
+    BATCH_MERGE_WITH_FETCHED_ENTRY(Localization.lang("Get bibliographic data from %0 (fully automated)", "DOI/ISBN/...")),
```

Comment:
Not sure, what I written in the last review, but I think, it was about consistency?

Haven't I written `MULTI_` be used as prefix?

![image](https://github.com/user-attachments/assets/bdb60e9b-725b-4ae8-a47c-9179f01cd037)